src/utils/templates.ts

```diff
@@ -68,7 +68,7 @@ export async function copyFiles(files: CopyDescriptor[], options: { [key: string
 function copyRemote(sourcePath: URL, targetFolder: string, fileName: string) {
   return new Promise((resolve, reject) => {
     https
-      .get(sourcePath, response => {
+      .get(sourcePath.toString(), response => {
```

Comment:
Maybe add a comment why this is necessary to avoid someone "fixing" it in the future

Reply:
I add a todo to remove it once the SDK version with the proper version is there.